.ci/aarch64/install_rom_aarch64.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+#
+# Copyright (c) 2021 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+source /etc/os-release || source /usr/lib/os-release
+
+EDK2_REPO="https://github.com/tianocore/edk2.git"
+EDK2_PLAT_REPO="https://github.com/tianocore/edk2-platforms.git"
+ACPICA="https://github.com/acpica/acpica.git"
+
+TOOLCHAIN_SOURCE_URL="https://armkeil.blob.core.windows.net/developer/Files/downloads/gnu-a/9.2-2019.12/binrel/gcc-arm-9.2-2019.12-x86_64-aarch64-none-elf.tar.xz"
+TOOLCHAIN_ARCHIVE="gcc-arm-9.2-2019.12-x86_64-aarch64-none-elf.tar.xz"
+TOOLCHAIN_PREFIX="gcc-arm-9.2-2019.12-x86_64-aarch64-none-elf/bin/aarch64-none-elf-"
+
+export WORKSPACE=$(mktemp -d)
+
+QEMU_EFI_BUILD_PATH="${WORKSPACE}/Build/ArmVirtQemu-AARCH64/RELEASE_GCC5/FV/QEMU_EFI.fd"
+QEMU_EFI_DEFUALT_PATH="/usr/share/qemu-efi-aarch64/QEMU_EFI.fd"
+
+PREFIX="${PREFIX:-/usr}"
+INSTALL_PATH="${DESTDIR:-}${PREFIX}/share/kata-containers"
+
+EFI_DEFAULT_PATH="/usr/share/qemu-efi-aarch64/QEMU_EFI.fd"
+
+FLASH0_NAME="kata-flash0.img"
+FLASH1_NAME="kata-flash1.img"
+
+arch=$(uname -m)
+
+build_uefi()
+{
+	cd "${WORKSPACE}"
+	git clone "${EDK2_REPO}"
+	git clone "${EDK2_PLAT_REPO}"
+	git clone "${ACPICA}"
+
+
+	sudo apt install -y python python3 python3-distutils uuid-dev build-essential bison flex
+
+	mkdir toolchain && cd toolchain
+	wget "${TOOLCHAIN_SOURCE_URL}" && tar -xf "${TOOLCHAIN_ARCHIVE}"
+	cd ..
+
+	make -C acpica/
+
+	export GCC5_AARCH64_PREFIX=$WORKSPACE/toolchain/gcc-arm-9.2-2019.12-x86_64-aarch64-none-elf/bin/aarch64-none-elf-
+	export PACKAGES_PATH=$WORKSPACE/edk2:$WORKSPACE/edk2-platforms
+	export IASL_PREFIX=$WORKSPACE/acpica/generate/unix/bin/
+
+	export PYTHON_COMMAND=/usr/bin/python
+	export PYTHON_COMMAND=/usr/bin/python3
+
+	cd edk2 && git submodule update --init && cd -
+	source edk2/edksetup.sh
+	make -C edk2/BaseTools
+
+	build -a AARCH64 -t GCC5 -p edk2/ArmVirtPkg/ArmVirtQemu.dsc -b RELEASE
+	echo "Info: cross build uefi successfully"
+}
+
+prepare_default_uefi()
+{
+	if [ "$ID" == "ubuntu" ] || [ "$ID" == "debian" ]; then
+		apt remove -y qemu-efi-aarch64
+		apt-get install -y qemu-efi-aarch64
+	else
+		local efi_path_dir="/usr/share/qemu-efi-aarch64/"
```

Comment:
This is the same as the dirname of `QEMU_EFI_DEFUALT_PATH` (aka `QEMU_EFI_DEFAULT_PATH`) and `EFI_DEFAULT_PATH`.